.github/workflows/Downgrade.yml

```diff
@@ -76,11 +76,11 @@ jobs:
           projects: ., test
       - uses: julia-actions/julia-buildpkg@v1
         env:
-          PYTHON: ""
+          PYTHON: ''
```

Comment:
Why this change?

Reply:
My default yaml format uses single quotes (other CI files also use single quotes, it is okay I think)